index.bs

```diff
@@ -523,7 +523,7 @@ The sample code for generating and registering a new key follows:
         }
       ],
 
-      timeout: 60000,  // 1 minute
+      timeout: 3600000,  // 6 minutes
```

Comment:
300000, // 5 minutes

Had recommended 5 minutes as default in PR #1317

Reply:
I know, but should we use the defaults as the example argument? It seems a bit strange to me since a better example of that would be to just leave the parameter out. I agree 5 minutes might be better for consistency, but I'm not sure which way to go on this...

Reply:
Actually you had 60 minutes in your example. :)
Either we leave the timeout or we add user verifcation required in above example. Both are fine with me.

Reply:
> Actually you had 60 minutes in your example. :)

Oh, oops! Thanks! :smile:

Adding UV seems like a good idea.